diff --git a/node_modules/snabbdom-to-html/init.js b/node_modules/snabbdom-to-html/init.js
index 2c00ba3..f6b7268 100644
--- a/node_modules/snabbdom-to-html/init.js
+++ b/node_modules/snabbdom-to-html/init.js
@@ -30,12 +30,12 @@ module.exports = function init (modules) {
       return ''
     }
 
-    if (!vnode.sel && typeof vnode.text === 'string') {
-      return escape(vnode.text)
+    if (!vnode.sel && (typeof vnode.text === 'string' || typeof vnode === 'string')) {
+      return escape(vnode.text || vnode)
     }
 
     vnode.data = vnode.data || {}
-
+// if(vnode.data.props) console.log(vnode)
     // Support thunks
     if (vnode.data.hook &&
       typeof vnode.data.hook.init === 'function' &&
